parts/linux/cloud-init/artifacts/cis.sh

```diff
@@ -173,25 +173,27 @@ fixUmaskSettings() {
     # it does no harm and works with the tools.
     # Note that we use printf to avoid a trailing newline.
     local umask_sh="/etc/profile.d/umask.sh"
-    if [[ "${OS}" == "${MARINER_OS_NAME}" && "${OS_VERSION}" == "2.0" && -f "${umask_sh}" ]]; then
-        printf "umask 027" >${umask_sh}
+    if [[ "${OS}" == "${MARINER_OS_NAME}" && "${OS_VERSION}" == "2.0" ]] || [[ "${OS}" == "${AZURELINUX_OS_NAME}" && "${OS_VERSION}" == "3.0" ]]; then
```

Comment:
This logic would fit better in a function to avoid duplication throughout the code

Reply:
Added a function `isMarinerOrAzureLinux` in cse_helpers.sh to reduce code duplication so that we do not have to say if Mariner or AzureLinux - [commit](https://github.com/Azure/AgentBaker/pull/4808/commits/a7e2a5f0ac56bedca7c5b5e065a58157c469a910#diff-f638b557a63aa8eb0f89db47ba718ac18b2af97dc6aed5ee97bfe1739cc58a48)